application/core/utils.py

```
@@ -197,3 +202,58 @@ def entity_attribute_sort_key(val, sort_order=ENTITY_ATTRIBUTE_ORDER):
+                            "elapsed_seconds": round(elapsed_time, 2),
+                            "function": func.__name__,
+                        },
+                    )
+                return result
+            except Exception as e:
+                elapsed_time = time.time() - start_time
+
+                if elapsed_time > threshold_seconds:
+                    # Log slow exception errors
+                    logger.error(
+                        f"Error in {func.__name__}: {str(e)}",
```

Ruff (0.15.10)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)